good-commit-message.md

```diff
@@ -3,6 +3,7 @@
 add your tip
 
 - Start a commit message with a verb eg. `add`, `change` etc
+- make it descriptive and not just a word
```

Comment:
Start with the capital letter when you start a sentence.

Reply:
Done